src/test/scala/nl.knaw.dans.easy.bagstore/server/StoresServletSpec.scala

```diff
@@ -366,9 +366,10 @@ class StoresServletSpec extends TestSupportFixture
   }
 
   private val basicAuthentication = authenticationHeader(username, password)
+  private val basicAuthenticationAndZipContentType = basicAuthentication  :+ "Content-Type" -> "application/zip"
 
   def putBag(uuid: String, bagZip: Path): Unit = {
-    put(s"/store1/bags/$uuid", body = Files.readAllBytes(bagZip), basicAuthentication) {
+    put(s"/store1/bags/$uuid", body = Files.readAllBytes(bagZip), basicAuthenticationAndZipContentType) {
```

Comment:
could you add `headers = ` to every one of these? just for clearance in reading.

```scala
put(s"/store1/bags/$uuid", body = Files.readAllBytes(bagZip), headers = basicAuthenticationAndZipContentType) {
```